1_fetch/src/get_nwis_data.R

```diff
@@ -0,0 +1,79 @@
+
+get_nwis_daily_data<-function(site_num,outdir,parameterCd,startDate, endDate){
+   ##read daily NWIS data, and save to csv file.
+  data_out<-readNWISdv(site_num, parameterCd, startDate, endDate)
+  
```

Comment:
Comment also applies to the peak flow data

Reply:
this is very odd.  I have changed the column names to discharge and discharge_cd.  I did a search for "X_" so hopefully when the names are a little wacky, they at least start with X_, otherwise this might not work.

Reply:
I'll take a look at how you programmed the check. I think the "cd" column will always be one to the right of the "discharge" column, so it could be `grep` the colnames for "cd" and take that column index as well as the index one smaller